gap/snksetswrsr.gi

```diff
@@ -474,12 +474,24 @@ RECOG.JellyHomFunc := function(data,el)
 end;
 
 #! @BeginChunk SnkSetswrSr
-#! This method tries to determine whether the input group <A>G</A> is acting
-#! primitively on <M>N</M> points, and
-#! is isomorphic to a large subgroup of <M>H \wr S_r</M> where <M>H</M> is
-#! <M>S_n</M> acting on <M>k</M>-sets and <M>N = {{n \choose k}}^r</M>
-#! and <M>kr > 1</M>. <Q>Large</Q> means that <A>G</A> contains a subgroup
-#! isomorphic to <M>A_n^r</M>. If <A>G</A> is imprimitive then the output is
+#! This method tries to determine whether the input group <A>G</A> is a
+#! large-base primitive group that neither is a symmetric nor an alternating
+#! group in its natural action.
+#! This method is an implementation of <Cite Key="LNPS06"/>.
+#!
+#! A primitive group <M>H</M> acting on <M>N</M> points is called large if
+#! there exist <M>n</M>, <M>k</M>, and <M>r</M> with
+#! <M>N = {{n \choose k}}^r</M>
+#! and up to a permutational isomorphism
+#! <M>H</M> is a subgroup of the product action wreath product
+#! <M>S_n \wr S_r</M>
```

Comment:
The line wrapping is really strange in this section. Usually I try to have lines < 78 columns wide, and then wrapped along those lines... It's OK to deviate, but here it really looks to me as if the text could be reformatted a bit to have slightly more evenly distributed line lengths...

Also, please add a comma here:
```suggestion
#! <M>S_n \wr S_r</M>,
```